data/json/mutations/mutation_category.json

```diff
@@ -66,6 +66,15 @@
     "iv_message": "Your pulse pounds as the waves.",
     "memorial_message": "Went deep."
   },
+  {
+    "type": "mutation_category",
+    "id": "GASTROPOD",
+    "name": "Gastropod",
+    "threshold_mut": "THRESH_GASTROPOD",
+    "mutagen_message": "Are you dancing a descending spiral?",
+    "iv_message": "The gaps between your body shrink.",
```

Comment:
?

Reply:
I'm open to suggestions.  the first I was making an oblique reference to spiral shells and the second to the PCs legs slowly merging into a single large "foot"

Reply:
The first one definitely works. I'll have to think about the second one.

Reply:
```suggestion
    "iv_message": "Your split lower body starts to come back together.",
```